src/main/java/com/bitpay/sdk/Client.java

```diff
@@ -329,6 +329,35 @@ public Invoice getInvoice(String invoiceId, String facade, Boolean signRequest)
 
         return invoice;
     }
+    
+    /**
+     * Retrieve a BitPay invoice by guid using the specified facade.  The client must have been previously authorized for the specified facade.
+     *
+     * @param guid        The guid of the invoice to retrieve.
+     * @param facade      The facade used to create it.
+     * @param signRequest Signed request.
+     * @return A BitPay Invoice object.
+     * @throws BitPayException       BitPayException class
+     * @throws InvoiceQueryException InvoiceQueryException class
+     */
+    public Invoice getInvoiceByGuid(String guid, String facade, Boolean signRequest) throws BitPayException, InvoiceQueryException {
```

Comment:
Would it make more sense use merchant if you have merchant, pos if not, and error if you have no facade token? As opposed to asking for the facade as parameter.